### PR TITLE
change the schedule so that the crontask is executed at least once an hour

### PR DESCRIPTION
**Pull request by sa2ajj**

### CODE REVIEW

**Comment by djmitche** [comment]
Looks good.

As an aside, at Mozilla we assign Puppet to run at a random minute past the half-hour on each host, where that "random minute" is stable from hour to hour on the host - using https://docs.puppetlabs.com/references/latest/function.html#fqdnrand.  It has the advantage of spacing puppet runs out evenly on each host, while not pounding the puppetmasters.  Maybe we could do something similar here.

**Comment by sa2ajj** [comment]
Funnily, this something I was looking for in Ansible.

I'll add a module to our `library`.

**Comment by sa2ajj** [comment]
@djmitche have a look at the filter I created and its usage.

File: jail-bot.yml [hunk 1]
```diff
@@ -7,4 +7,3 @@
   sudo: yes
   roles:
   - base
-
```
Comment (djmitche) [inline jail-bot.yml]: Aww, i spent a lot of time writing that trailing newline.

File: library/bb_filters.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+from hashlib import new as new_hash
+from random import Random
+
+def seeded_range(seed, start, stop=None, step=1, extra=None):
+    """
+    a filter of sort
```
Comment (djmitche) [inline library/bb_filters.py]: Can you update this so it's clear how to use it?

**Comment by djmitche** [comment]
Looks great!

File: library/bb_filters.py (hunk 2 above)
Comment (sa2ajj) [inline library/bb_filters.py]: he-he. I meant to do that but forgot :)

**Comment by sa2ajj** [comment]
I feel stuck re docs.  Do you think this will be enough?

**Comment by djmitche** [comment]
Yup, that's great

**Comment by sa2ajj** [comment]
@djmitche I checked a couple of hosts and it works as expected! Yay :)

**Pull request merged**